app/models/agents/post_agent.rb

```diff
@@ -124,15 +124,15 @@ def receive(incoming_events)
       incoming_events.each do |event|
```

Comment:
Inserting `interpolate_with(event) do` here eliminates the need for passing `event` or `event.payload` to `interpolated()`, and that's how some other agents do to pull in each event payload to Liquid evaluation.